client/blocks/post-actions/index.jsx

```diff
@@ -109,6 +122,7 @@ PostActions.propTypes = {
 
 const mapStateToProps = ( state, { siteId, post } ) => {
 	const isJetpack = isJetpackSite( state, siteId );
+	const siteDomain = getSiteDomain( state, siteId );
```

Comment:
I've been using [`getSiteSlug`](https://github.com/Automattic/wp-calypso/blob/master/client/state/sites/selectors.js#L222-L240) in the Comment redesign to obtain the site fragment and build URLs.

Is there a reason to choose `getSiteDomain` instead?

Reply:
Looking at the code it's somewhat confusing. `getSiteDomain` in certain cases uses `getSiteSlug`. But getSiteSlug is guaranteed to pass use `urlToSlug`.

Maybe let's stick with `getSiteSlug`

Reply:
Wow, that _is_ confusing (I went with `getSiteDomain` as searching "domain" was the first thing I thought of when wanting, uh, the domain). I'll update to `getSiteSlug`.